OpenRA.Game/Map/CellLayerBase.cs

```diff
@@ -35,6 +35,11 @@ public CellLayerBase(MapGridType gridType, Size size)
 			entries = new T[size.Width * size.Height];
 		}
 
+		public virtual void Clear()
```

Comment:
Can we move this down next to `public void Clear(T clearValue)` and maybe change that to use `Array.Fill` to match?

Reply:
Done. This made me realise that `Clear(T clearValue)` needed the same safety check to ensure the `CellEntryChanged` event was not in use if called. Have added that in.